localisation/00_wc_artifact_localisation.csv

```diff
@@ -0,0 +1,100 @@
+remnant_of_caer_darrow;Remnant of Caer Darrow;;�berreste von Caer Darrow;;;;;;;;;;;
+ring_of_kaldorei_empire;Ring of the Kaldorei Empire;;Ring des Kaldorei-Imperiums;;;;;;;;;;;
+ring_of_kaldorei_empire_desc;;;;;;;;;;;;;;
+ring_of_trollbanes;Trollbane Sigil Ring;;Siegelring der Trollbanes;;;;;;;;;;;
+ring_of_trollbanes_desc;A golden ring with the richest gems, this ring is a direct proof of the incredible wealth of the Trollbane family, which they earned over many centuries of owning a multitude of mines in the Arathi Highlands.;;Dieser goldene Ring mit den reichsten Edelsteinen ist ein direkter Beweis f�r den unglaublichen Reichtum der Trollbane-Familie, den sie sich �ber viele Jahrhunderte hinweg durch den Besitz einer Vielzahl von Minen im Arathi-Hochland erworben haben.;;;;;;;;;;;
+runeblade_blood_3;Blood Runeblade;;Runenklinge des Bluts;;;;;;;;;;;
+runeblade_frost_3;Frost Runeblade;;Runenklinge des Frosts;;;;;;;;;;;
+runeblade_unholy_3;Unholy Runeblade;;Unheilige Runenklinge;;;;;;;;;;;
+;;;;;;;;;;;;;;
+# Uncommon;;;;;;;;;;;;;;
+cloak_of_trollbanes;Mantle of Stromgarde;;Mantel von Stromgarde;;;;;;;;;;;
+cloak_of_trollbanes_desc;This mantle used to be a part of the once great Arathian Empire's flag, which the ancestors of the Trollbanes' preserved and wore as a cloak after the empire's fall.;;Dieser Mantel war fr�her Teil der einst gro�en Flagge des arathorischen Reiches, die von den Vorfahren der Trollbanes bewahrt und nach dem Fall des Reiches als Umhang getragen wurde.;;;;;;;;;;;
+gauntlets_of_trollbanes;Ignaeus' Gauntlets;;Ignaeus Handschuhe;;;;;;;;;;;
+gauntlets_of_trollbanes_desc;A family relic of the Trollbane dynasty, which were worn by many outstanding generals of this family. Ever since Ignaeus Trollbane, the ancestor of the Trollbane dynasty and the great general of the Arathorian Empire, military education has been a family pride.;;Eine Familienreliquie aus der Trollbane-Dynastie, die von vielen hervorragenden Gener�len dieser Familie getragen wurde. Seit Ignaeus Trollbane, dem Stammvater der Trollbane-Dynastie und dem gro�en General des arathorischen Reiches, ist die milit�rische Bildung ein Familienstolz.;;;;;;;;;;;
+libram_of_devoutness_02;Libram of Devoutness;;Foliant der Fr�mmigkeit;;;;;;;;;;;
+libram_of_guardianship_02;Libram of Guardianship;;Foliant der Bewahrung;;;;;;;;;;;
+libram_of_reckoning_02;Libram of Reckoning;;Foliant der Abrechnung;;;;;;;;;;;
+libram_of_righteousness_02;Libram of Righteousness;;Foliant der Rechtschaffenheit;;;;;;;;;;;
+runeblade_blood_2;Blood Runeblade;;Runenklinge des Bluts;;;;;;;;;;;
+runeblade_frost_2;Frost Runeblade;;Runenklinge des Frosts;;;;;;;;;;;
+runeblade_unholy_2;Unholy Runeblade;;Unheilige Runenklinge;;;;;;;;;;;
+;;;;;;;;;;;;;;
+# Common;;;;;;;;;;;;;;
```

Comment:
When you asked about sorting in discord, I warned you that new localization strings always will be added at the bottom of localization file and there is no point in sorting them.

Reply:
but it doesn't hurt to start with a sorted file, does it?

Reply:
Well, until someone will misunderstood how its works and attempt to sort new lines